src/main/java/de/craften/ui/swingmaterial/MaterialColor.java

```diff
@@ -279,6 +279,7 @@
     public static final Color GREY_600 = Color.decode("#757575");
     public static final Color GREY_700 = Color.decode("#616161");
     public static final Color GREY_800 = Color.decode("#424242");
+    public static final Color GREY_850 = Color.decode("#313131");
```

Comment:
See the [Themes section](https://material.io/guidelines/style/color.html#color-themes) in the Material Design Color Guide. Dark UIs use a color between 900 and 800 for the default content area (#3). `MaterialColor.GREY_850` is meant to enact as such color in order to avoid unnecessary hardcoding.

Now that I see, maybe `MaterialWindow.light()` should set `MaterialColor.GREY_50` as background instead of `MaterialColor.WHITE`.